V2.16-RUN-15152-alertmanager-alerts (#633)
run-ai/docs: docs/admin/troubleshooting/alertmanager/RunaiContainerMemoryUsageCritical.md

---
title: RunaiContainerMemoryUsageCritical
summary: This article describes the RunaiContainerMemoryUsageCritical alert.
authors:
    - Jason Novich
    - Viktor Koukouliev
date: 2024-Jan-10
---

## Meaning

> The `runai` container is using more than 90% of its Memory limit.

Reviewer: ```suggestion
A Run:ai container is using more than 90% of its Memory limit.
```

## Impact

The container might go out of memory (OOM) and crash.

## Severity

Critical

## Diagnosis

> Use <!-- add a command here --> on the relevant pod.  

Reviewer: ```suggestion
Use the command `kubectl top` on the relevant pod specified in the alert.  
```

If this tool is unavailable, you can calculate the memory usage by running `container_memory_usage_bytes{namespace=~"runai|runai-backend"}`.

## Mitigation

> Add memory resources to the container. If the issue does not resolve, contact Run:ai.

Reviewer: ```suggestion
Add memory resources to the container. 
If the issue is not resolved, contact Run:ai support.
```
